sys/include/can/can.h

```diff
@@ -101,7 +101,7 @@ struct can_frame {
 struct can_filter {
     canid_t can_id;    /**< CAN ID */
     canid_t can_mask;  /**< Mask */
-#if defined(MODULE_MCP2515)
+#if !defined(__linux__)
```

Comment:
since this is a special feature of some can devices this can't be a black list of drives that don't support this but must be a whitelist of driver that do.

Reply:
Only native boards are not supporting the target mailbox currently, it is easier to then just exclude it than include other CAN controllers here

Reply:
might be easier but wrong you should only use features that u have some kind of clue for that they exist not the other way around

Reply:
or something like : '#define HAVE_CAN_FILTER_MAILBOX'